BasiliskII/src/SDL/video_sdl.cpp

```diff
@@ -1025,6 +1033,11 @@ bool VideoInit(bool classic)
 	else if (default_height > sdl_display_height())
 		default_height = sdl_display_height();
 
+	// for classic Mac, make sure the display width is 8
+	if (classic) {
+		default_width = (default_width << 3) >> 3;
```

Comment:
Nit: (default_width / 8) * 8

No reason to use shifts since compilers will generate the same instructions anyway.